src/scss/abstracts/_mixins.scss

```diff
@@ -1 +1,46 @@
 // Mixins.
+@mixin uio-focus-style($c) {
+	a[href="https://www.ic.gc.ca/eic/site/118.nsf/eng/home"],
```

Comment:
The generic styles that don't need to adjust to `$fColor` can be defined outside this function. This block is one of them as well as styling blocks at 22, 29.